Sources/SecureXPC/Client/XPCMachClient.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  XPCMachClient.swift
+//  SecureXPC
+//
+//  Created by Josh Kaplan on 2021-10-09
+//
+
+import Foundation
+
+internal class XPCMachClient: XPCClient {
+	/// Creates a client which will attempt to send messages to the specified mach service.
+	///
+	/// - Parameters:
+	///   - machServiceName: The name of the XPC mach service; no validation is performed on this.
+	public init(machServiceName: String) {
```

Comment:
Is there a reason to have this initializer? It should inherit `XPCClient`'s and since all this is doing is calling that one, don't believe this serves a purpose.

Reply:
Nope! ✂️